@xen-orchestra/lite/src/locales/en.json

```
@@ -45,13 +45,15 @@
   "password": "Password",
   "password-invalid": "Password invalid",
   "pool-cpu-usage": "Pool CPU Usage",
+  "pool-ram-usage": "Pool RAM Usage",
   "property": "Property",
   "ram-usage":"RAM usage",
   "send-us-feedback": "Send us feedback",
   "settings": "Settings",
   "snapshot": "Snapshot",
   "sort-by": "Sort by",
   "stacked-cpu-usage":"Stacked CPU usage",
+  "stacked-ram-usage": "Stacked memory usage",
```

Since we use `Pool CPU Usage`, `Pool RAM Usage` and `Stacked CPU usage`, shouldn't this one to be `Stacked RAM usage` ?

(I know its from Figma, but maybe can we keep consistency here? @pdonias @MathieuRA ?)

```suggestion
  "stacked-ram-usage": "Stacked RAM usage",
```

---

It makes sense, but IMO, if we change something, we should at least leave a comment on Figma to make sure we all agree and the mockup can be updated.

---

I'm waiting for Clémence's response on that. I can change it in another PR.